README.md

```diff
@@ -70,9 +76,11 @@ The config passed to `Configure` can contain the following fields.
 | `bucket`          | the GCS bucket name                                                                 | yes       | "bucket_name"       |
 | `pollingPeriod`       | polling period for the CDC mode, formatted as a time.Duration string. default is "1s"  | no        | "2s", "500ms"       |
 
-
+When testing from the swagger or any rest client provide the stringified value in `serviceAccountKey`
+Example: console.log(JSON.stringify(JSON.stringify(key file content))) run in the JavaScript to get the preferred stringified value because wrapping JSON around the single quotes is not accepted in swagger OR
```

Comment:
```suggestion
Example: `console.log(JSON.stringify(JSON.stringify(key file content)))` run this JavaScript command to get the preferred stringified value. because wrapping JSON with single quotes is not accepted in swagger OR
```

Reply:
Thanks its Done